Translators/WZDx/main.py

```diff
@@ -31,6 +31,7 @@ def update_rsu_region_name(request, tim_body):
 
 def translate(wzdx_geojson):
     tims = []
+    duration = os.getenv("DURATION_TIME")

```

Comment:
Let's add this variable to the README to document it. Let's also add in a default value to fall back on just in case it isn't set here.

Reply:
Added a default of 1800 seconds and added DURATION_TIME to the README

Reply:
Added  DURATION_TIME to README and added default value of 30 minute duration time